### ISSUE TITLE
Error when set log level to debug

### ISSUE THREAD

**Issue by zean00** [motiv-labs/janus#267]
Error occurred when set log level to debug through configuration

**Reproduction Steps:**

1. Change 
```
[log] 
  level = "debug"
```
2. Start janus
3. Error occurred 
```
fatal error: all goroutines are asleep - deadlock!
apigw_1         | 
apigw_1         | goroutine 1 [semacquire]:
apigw_1         | sync.runtime_SemacquireMutex(0xc42005e1c0, 0x1ba056f72010e000)
apigw_1         | 	/usr/local/go/src/runtime/sema.go:71 +0x3d
apigw_1         | sync.(*Mutex).Lock(0xc42005e1bc)
apigw_1         | 	/usr/local/go/src/sync/mutex.go:134 +0xee
apigw_1         | github.com/hellofresh/janus/vendor/github.com/sirupsen/logrus.(*MutexWrap).Lock(0xc42005e1bc)
apigw_1         | 	/go/src/github.com/hellofresh/janus/vendor/github.com/sirupsen/logrus/logger.go:43 +0x3d
apigw_1         | github.com/hellofresh/janus/vendor/github.com/sirupsen/logrus.Entry.log(0xc42005e190, 0xc4203e6de0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x5, ...)
apigw_1         | 	/go/src/github.com/hellofresh/janus/vendor/github.com/sirupsen/logrus/entry.go:97 +0x103
apigw_1         | github.com/hellofresh/janus/vendor/github.com/sirupsen/logrus.(*Entry).Debug(0xc4203ec410, 0xc4203376d8, 0x1, 0x1)
apigw_1         | 	/go/src/github.com/hellofresh/janus/vendor/github.com/sirupsen/logrus/entry.go:134 +0xac
apigw_1         | github.com/hellofresh/janus/vendor/github.com/hellofresh/stats-go/incrementer.(*Log).Increment(0x11913f8, 0xc4203aaba0, 0x13)
apigw_1         | 	/go/src/github.com/hellofresh/janus/vendor/github.com/hellofresh/stats-go/incrementer/log.go:13 +0xbc
apigw_1         | github.com/hellofresh/janus/vendor/github.com/hellofresh/stats-go.(*StatsdClient).TrackMetric(0xc4203e69f0, 0xc0f279, 0x9, 0xc03772, 0x5, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
apigw_1         | 	/go/src/github.com/hellofresh/janus/vendor/github.com/hellofresh/stats-go/statsd_client.go:110 +0xd1
apigw_1         | github.com/hellofresh/janus/vendor/github.com/hellofresh/stats-go/hooks.(*LogrusHook).Fire(0xc42038dcc0, 0xc4203ec370, 0xc400000002, 0xc42008c6e8)
apigw_1         | 	/go/src/github.com/hellofresh/janus/vendor/github.com/hellofresh/stats-go/hooks/logrus.go:27 +0xc8
apigw_1         | github.com/hellofresh/janus/vendor/github.com/sirupsen/logrus.LevelHooks.Fire(0xc42000e5a0, 0x2, 0xc4203ec370, 0xc4200e4100, 0x10000c42010e000)
apigw_1         | 	/go/src/github.com/hellofresh/janus/vendor/github.com/sirupsen/logrus/hooks.go:28 +0x99
apigw_1         | github.com/hellofresh/janus/vendor/github.com/sirupsen/logrus.Entry.log(0xc42005e190, 0xc4203a62d0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x2, ...)
apigw_1         | 	/go/src/github.com/hellofresh/janus/vendor/github.com/sirupsen/logrus/entry.go:98 +0x12b
apigw_1         | github.com/hellofresh/janus/vendor/github.com/sirupsen/logrus.(*Entry).Error(0xc42005f8b0, 0xc420337a80, 0x1, 0x1)
apigw_1         | 	/go/src/github.com/hellofresh/janus/vendor/github.com/sirupsen/logrus/entry.go:160 +0xac
apigw_1         | github.com/hellofresh/janus/vendor/github.com/sirupsen/logrus.(*Logger).Error(0xc42005e190, 0xc420337a80, 0x1, 0x1)
apigw_1         | 	/go/src/github.com/hellofresh/janus/vendor/github.com/sirupsen/logrus/logger.go:219 +0x6e
apigw_1         | github.com/hellofresh/janus/vendor/github.com/sirupsen/logrus.Error(0xc420337a80, 0x1, 0x1)
apigw_1         | 	/go/src/github.com/hellofresh/janus/vendor/github.com/sirupsen/logrus/exported.go:102 +0x4b
apigw_1         | main.initDatabase()
apigw_1         | 	/go/src/github.com/hellofresh/janus/cmd/janus/init.go:121 +0xc9
apigw_1         | main.RunServer(0x1163660, 0xc42038cd60, 0x0, 0x2)
apigw_1         | 	/go/src/github.com/hellofresh/janus/cmd/janus/server.go:51 +0xe3
apigw_1         | github.com/hellofresh/janus/vendor/github.com/spf13/cobra.(*Command).execute(0x1163660, 0xc42000e0a0, 0x2, 0x2, 0x1163660, 0xc42000e0a0)
apigw_1         | 	/go/src/github.com/hellofresh/janus/vendor/github.com/spf13/cobra/command.go:750 +0x2c1
apigw_1         | github.com/hellofresh/janus/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x1163660, 0x4046a4, 0xc420058058, 0x0)
apigw_1         | 	/go/src/github.com/hellofresh/janus/vendor/github.com/spf13/cobra/command.go:831 +0x30e
apigw_1         | github.com/hellofresh/janus/vendor/github.com/spf13/cobra.(*Command).Execute(0x1163660, 0x28, 0x0)
apigw_1         | 	/go/src/github.com/hellofresh/janus/vendor/github.com/spf13/cobra/command.go:784 +0x2b
apigw_1         | main.main()
apigw_1         | 	/go/src/github.com/hellofresh/janus/cmd/janus/main.go:34 +0x31
```
**Expected behavior:**

Should not fatal error

**Observed behavior:**

Working fine when log level set back to info

**Janus version:** 
Last git commit
**OS and version:** 
Linux (docker)

**Comment by italolelis** [comment]
@zean00 Thank you for the report. I tried to reproduce this error but wasn't able to. Would you mind providing your whole configuration file?

**Comment by zean00** [comment]
Hi

It seems that it already fixed with the release 3.6.0, it was happened with release 3.5.0. Below is my configuration

```
################################################################
# Global configuration
################################################################
port = 8080
#logLevel = "info"
#debug = true

[log]
	level = "debug"
################################################################
# API configuration backend
################################################################
[web]
  port = 8081

  [web.credentials]
    secret = "secret"

    [web.credentials.basic]
    users = {admin = "admin"}

[database]
	dsn = "file:///etc/janus"

[storage]
	dsn = "memory://localhost"

[tracing]
	provider = "jaeger"

[tracing.jaeger]
	dsn = "jaeger:5775"
	serviceName = "apigw"
```

And here is my Dockerfile for 3.5.0 release

```
FROM golang:1.9-alpine AS builder

WORKDIR /go/src/github.com/hellofresh/janus
RUN apk add --update bash make git
RUN cd /go/src/github.com/hellofresh && \
	git clone https://github.com/hellofresh/janus.git && \
	cd /go/src/github.com/hellofresh/janus && \
	git checkout tags/3.5.0 && \
	make

# ---

FROM alpine

#ADD assets/ca-certificates.crt /etc/ssl/certs/
COPY --from=builder /go/src/github.com/hellofresh/janus/dist/janus_linux-amd64 /

RUN mkdir -p /etc/janus/apis && \
    mkdir -p /etc/janus/auth

RUN apk add --update curl && \
    rm -rf /var/cache/apk/*

EXPOSE 8080 8081 8443 8444
ENTRYPOINT ["/janus_linux-amd64"]
```
